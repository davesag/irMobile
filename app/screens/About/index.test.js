/**
 * @format
 */

import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import AboutScreen from '.'

let tree

describe('rendering', () => {
  const navigation = {}

  beforeAll(() => {
    tree = renderer.create(<AboutScreen navigation={navigation} />)
  })

  it('rendered correctly', () => {
    expect(tree).toMatchSnapshot()
  })
})
