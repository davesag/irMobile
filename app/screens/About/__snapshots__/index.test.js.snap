// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering rendered correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "stretch",
      "flex": 1,
      "justifyContent": "flex-start",
      "margin": 10,
    }
  }
>
  <View />
</RCTSafeAreaView>
`;
